### PR TITLE
chore(docs): add cawemo EOL announcement

### PR DESCRIPTION
**Pull request by marcellobarile**
Closes https://github.com/camunda/cawemo/issues/11601

NOTE: The SaaS announcement has been agreed with @crobbins215 ; the On-premises announcement has to be peer-reviewed completely. I'm not sure if what I put there is enough.

### CODE REVIEW

**Comment by marcellobarile** [comment]
This is how it will look like:
## SaaS:
image

## On-premises:
image

File: enterprise/content/announcement.md [hunk 1]
```diff
@@ -612,6 +612,10 @@ __Changes in Supported Environments:__
 
 # Cawemo On-Premises Announcements
 
+## End of Life Notification
+
+**Effective 30th April 2027**
```
Comment (marcellobarile) [inline enterprise/content/announcement.md:617]: I think that @CatalinaMoisuc suggested showing other info under the `on-premises` announcement; e.g.:
`whereas Cawemo On-Premises will continue to be maintained as per the C7 maintenance timeline 2027 and beyond that (max 2030) only on exceptional cases, with costs attached (if I remember correctly)`

I might need your input and help to construct such a content.

**Review by crobbins215 — COMMENTED** [review]
Saas looks good; on-premises I've added a comment for review.

File: enterprise/content/announcement.md [hunk 2]
```diff
@@ -612,6 +612,10 @@ __Changes in Supported Environments:__
 
 # Cawemo On-Premises Announcements
 
+## End of Life Notification
+
+Cawemo On-Premises will continue to receive support, aligned with the Camunda 7 extended support timelines. For detailed information on these timelines and the support scope, please refer to the [Camunda 7 Extended Support Offering](https://docs.camunda.org/enterprise/announcement/#camunda-extended-support-offering).
```
Comment (marcellobarile) [inline enterprise/content/announcement.md:617]: @crobbins215 here we go :)

Comment (crobbins215) [reply enterprise/content/announcement.md:617]: Yep, that's good

**Comment by marcellobarile** [comment]
> Both messages align with our customer messaging, which I'm happy with. Are we now waiting for either @akeller or @christinaausley for review to approve from docs point of view?

I think so, yes.

**Comment by marcellobarile** [comment]
Hi!
Just checking in on the status of the review. Whenever you get a chance, your feedback would be much appreciated. Let me know if there's anything specific you'd like me to address. Thanks! :)

File: enterprise/content/announcement.md [hunk 3]
```diff
@@ -675,8 +679,33 @@ __Release Date:__ 13th of November 2019
...
+
+**Effective 30th April 2024**
+
+As part of our ongoing commitment to enhance collaboration between Business and IT teams, we at Camunda have been diligently working to evolve our product offerings.
+
+In line with this vision and based on insightful customer conversations, we have already combined the power of Cawemo and our Desktop Modeler to create Web Modeler, which is the successor to Cawemo. The Web Modeler is designed to significantly improve time to value for both developers and business users by fostering a unified environment that enhances collaboration.
+
+**Key Transition Information:**
+
+End of Life for Cawemo SaaS: Effective 30th April 2024, the Cawemo SaaS service will be retired for all customers and users. This decision aligns with our strategic focus on delivering an integrated, comprehensive modeling experience.
```
Comment (christinaausley) [inline enterprise/content/announcement.md:694]: @akeller Does this mean we will need to remove https://docs.camunda.io/docs/next/guides/migrating-from-cawemo/ at some point? Or can retain as-is, with the note that Cawemo has an EOL date?

File: enterprise/content/announcement.md [hunk 4]
```diff
@@ -675,8 +679,33 @@ __Release Date:__ 13th of November 2019
...
+
+Data Access and Transition Deadline: This means that you have until the 30th of April to export your data and transition to one of the options outlined below. Please be advised that after the 30th of April, you will not be able to access data within Cawemo SaaS. In line with our data retention policy, your data will be stored for an additional 30 days after this date. It is crucial to export any required data before the 30th of April to ensure you retain access to your information.
+
+**Transition Options:**
```
Comment (christinaausley) [inline enterprise/content/announcement.md:698]: I would provide links to documentation on both of these options.

File: enterprise/content/announcement.md (hunk 3 above)
Comment (crobbins215) [reply enterprise/content/announcement.md:694]: https://docs.camunda.io/docs/next/guides/migrating-from-cawemo/ needs to stay until after the April deadline, we should just add the Cawemo EOL date

Comment (akeller) [reply enterprise/content/announcement.md:694]: @christinaausley 100% agree with @crobbins215 here. Can you assist him with a PR on docs.camunda.io? We should also add an issue to our DevEx backlog to remove the guide in the future.

**Comment by crobbins215** [comment]
@marcellobarile, I've made an adjustment to the messaging for further clarity. Are you able to adjust this section?

> Transition Options:
> Move to our Web Modeler: Experience the combined power of Cawemo and our Desktop Modeler in a seamless, collaborative environment. For modeling only or Camunda 8 processes.
> 
> 
Move to our Desktop Modeler: Opt for a standalone BPMN modeling experience. Support both Camunda 7 and Camunda 8 processes. 
> 

> Move to Cawemo On-Premise: Continue with the familiar Cawemo environment, For modeling only or Camunda 7 processes. Please note that support for Cawemo On-Premise aligns with [Camunda 7 timelines.](https://docs.camunda.org/enterprise/announcement/)

File: enterprise/content/announcement.md [hunk 5]
```diff
@@ -675,8 +679,33 @@ __Release Date:__ 13th of November 2019
...
+
+Move to our Web Modeler: Experience the combined power of Cawemo and our Desktop Modeler in a seamless, collaborative environment. For modeling only or Camunda 8 processes.
```
Comment (marcellobarile) [inline enterprise/content/announcement.md:700]: @crobbins215 this has been updated accordingly to your latest input.

File: enterprise/content/announcement.md (hunk 4 above)
Comment (marcellobarile) [reply enterprise/content/announcement.md:698]: @christinaausley I might need the exact word/phrase to link and possibly also the available links. Thanks! 🤗

File: enterprise/content/announcement.md (hunk 5 above)
Comment (crobbins215) [reply enterprise/content/announcement.md:700]: Thank you

File: enterprise/content/announcement.md (hunk 4 above)
Comment (christinaausley) [reply enterprise/content/announcement.md:698]: For example:

- Move to [Web Modeler](/docs/components/modeler/web-modeler/new-web-modeler.md): Experience the combined power of Cawemo and our Desktop Modeler in a seamless, collaborative environment. For modeling only or Camunda 8 processes.
- Move to [Desktop Modeler](/components/modeler/desktop-modeler/install-the-modeler.md): Opt for a standalone BPMN modeling experience. Support both Camunda 7 and Camunda 8 processes.

Comment (marcellobarile) [reply enterprise/content/announcement.md:698]: @christinaausley I've added the links as suggested. Could you have a quick look?

**Review by crobbins215 — APPROVED** [review]
Checked links and messaging, matches communication

File: enterprise/content/announcement.md (hunk 4 above)
Comment (marcellobarile) [reply enterprise/content/announcement.md:698]: @christinaausley it looks like the two URLs provided here are wrong once deployed. Do you have any clue?

Comment (christinaausley) [reply enterprise/content/announcement.md:698]: You are linking from C7 to C8 docs across repos, so you'll have to use https://docs.camunda.io/docs/next/components/modeler/web-modeler/new-web-modeler/, for example.

**Pull request merged**